types/siren-parser/superagent.d.ts

```diff
@@ -0,0 +1,6 @@
+import { Action, Entity } from './index';
+import { SuperAgentRequest, Response } from 'superagent';
+
+export function parse(res: string): Entity;
+export function parse(res: Response, fn: (err: Error | null, body: Entity) => void): void;
+export function perform(request: any, action: Action): SuperAgentRequest;
```

Comment:
It looks the SuperAgentRequest needs to be augmented here to have the `submit` method:
https://github.com/Brightspace/node-siren-parser/blob/master/src/superagent.js#L37
Otherwise sample code from Readme won't work:
https://github.com/Brightspace/node-siren-parser/blob/master/README.md#L88

Reply:
I did notice that while writing the test code, but the `submitHelper` function isn't exported and it's not referenced inside that file, so I'm not sure how that's running. Could that be a "bug" (in the docs)?